From e73753f587d53833b6e35c85b29e8526536b3175 Mon Sep 17 00:00:00 2001
From: Lu Hui <luhui@sipeed.com>
Date: Wed, 23 Aug 2023 14:01:36 +0800
Subject: [PATCH 2/2] include: configs: light-c910.h: change bootpartition

---
 include/configs/light-c910.h | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/include/configs/light-c910.h b/include/configs/light-c910.h
index 63f99ef5..ecdcee5a 100644
--- a/include/configs/light-c910.h
+++ b/include/configs/light-c910.h
@@ -138,15 +138,16 @@
 	"kdump_buf=180M\0" \
 	"mmcdev=0\0" \
 	"mmcbootpart=2\0" \
-	"boot_conf_file=/extlinux/extlinux.conf\0" \
+	"mmcrootpart=3\0" \
+	"boot_conf_file=/boot/extlinux/extlinux.conf\0" \
 	"uuid_rootfsA=80a5a8e9-c744-491a-93c1-4f4194fd690a\0" \
 	"partitions=name=table,size=2031KB;name=boot,size=500MiB,type=boot;name=root,size=-,type=linux,uuid=${uuid_rootfsA}\0" \
 	"gpt_partition=gpt write mmc ${mmcdev} $partitions\0" \
-	"load_aon=ext4load mmc ${mmcdev}:${mmcbootpart} $fwaddr light_aon_fpga.bin;cp.b $fwaddr $aon_ram_addr $filesize\0" \
-	"load_c906_audio=ext4load mmc ${mmcdev}:${mmcbootpart} $fwaddr light_c906_audio.bin;cp.b $fwaddr $audio_ram_addr $filesize\0" \
+	"load_aon=load mmc ${mmcdev}:${mmcbootpart} $fwaddr light_aon_fpga.bin;cp.b $fwaddr $aon_ram_addr $filesize\0" \
+	"load_c906_audio=load mmc ${mmcdev}:${mmcbootpart} $fwaddr light_c906_audio.bin;cp.b $fwaddr $audio_ram_addr $filesize\0" \
 	"finduuid=part uuid mmc ${mmcdev}:${mmcpart} uuid\0" \
-	"bootcmd_load=run findpart;run load_aon;run load_c906_audio; ext4load mmc ${mmcdev}:${mmcbootpart} $opensbi_addr fw_dynamic.bin\0" \
-	"bootcmd=run bootcmd_load; bootslave; sysboot mmc ${mmcdev}:${mmcbootpart} any $boot_conf_addr_r $boot_conf_file;\0" \
+	"bootcmd_load=run findpart;run load_aon;run load_c906_audio; load mmc ${mmcdev}:${mmcbootpart} $opensbi_addr fw_dynamic.bin\0" \
+	"bootcmd=run bootcmd_load; bootslave; sysboot mmc ${mmcdev}:${mmcrootpart} any $boot_conf_addr_r $boot_conf_file;\0" \
 	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
 	"\0"
 
-- 
2.41.0

